Amazon.Ion.ObjectMapper/IonSerializer.cs

```diff
@@ -101,24 +101,40 @@ private string FullName(string typeName, string assemblyName)
     public class IonSerializationOptions
     {
         public IonPropertyNamingConvention NamingConvention { get; init; } = new CamelCaseNamingConvention();
+

```

Comment:
Why all the additions of whitespace - please revert for now until we apply styling.

Reply:
Fixed